src/foam/nanos/u2/navigation/UserInfoNavigationView.js

```diff
@@ -70,17 +70,31 @@ foam.CLASS({
                 .add( this.subject.realUser.toSummary() )
               .end();
         }));
-      var menuArray = (await menu.children.orderBy(foam.nanos.menu.Menu.ORDER).select())
-        .array.map( menuItem => {
-          var e = this;
-          return self.Action.create({
-            name: menuItem.name,
-            label: this.translationService.getTranslation(foam.locale, menuItem.id + '.label', menuItem.label),
-            code: () => {
-              menuItem.launch_(X, e);
-            }
-          });
+
+      // We need to add menus from settings (and then add menus from theme.settingsRootMenu) 
+      // because some menus are used in both settings and theme.settingsRootMenu (e.g., sign-out).
+      // Doing this prevents us from creating the same menu for each setting.
+      let menu = this.Menu.create({ id: 'settings' });
+      let menuArray = (await menu.children.select()).array;
+      
+      // add theme.settingsRootMenu menus
+      if ( this.theme.settingsRootMenu !== 'settings' ) {
```

Comment:
Is this a non-foam feature? 'settings'?

Reply:
I thought it was a foam feature. @Nauna Can you please confirm?

Reply:
settings is foam side, so all good.